### ISSUE TITLE
WinUI 3.0a: SolidColorBrush uses Windows.UI.Color types

### ISSUE THREAD

**Issue by dotMorten** [microsoft/microsoft-ui-xaml#1541]
**Describe the bug**
It seems weird to me that the Color property of the SolidColorBrush is the `Windows.UI.Color` WinRT type and not `Microsoft.UI.Color`.
(Also it would be nice with implicit conversion to/from `System.Drawing.Color`)

**Steps to reproduce the bug**
Inspect the Color type on SolidColorBrush.

**Expected behavior**
Don't rely on a WinRT type.

**Version Info**

NuGet package version: 3.0 Alpha 

**Additional context**

**Comment by MikeHillberg** [comment]
The thinking on this is that Color isn't likely to change, and it's used for more than just Windows.UI. For example it's used in [MediaClip.CreateFromColor](https://docs.microsoft.com/uwp/api/Windows.Media.Editing.MediaClip.CreateFromColor) and [DataPackagePropertySet.LogoBackgroundColor](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.DataTransfer.DataPackagePropertySet.LogoBackgroundColor). It's closer to a Windows.Foundation type like Rect than a WinUI type.

Colors and ColorHelper on the other hand, also in Windows.UI, are static classes, so not being used as an exchange type, and are more likely to change. So those were brought into the Microsoft.UI namespace.

**Comment by jevansaks** [comment]
I was feeling ok about the ABI-level types being Windows.UI.* but that was when I thought that the projections would all map it to local types. For cases like this I wonder if we should reconsider -- .NET does have a pretty well-established Color type that we should maybe map to and it would avoid this oddity.

**Comment by dotMorten** [comment]
I don't really get that color is like the rect type. IMHO that should move too then.

**Comment by MikeHillberg** [comment]
Rect and Color are both referenced by types outside of Windows.UI. They're broader in scope than just WinUI. So we could create a new Color type, but I don't see the benefit to it. (There are other types that Xaml references that aren't in WinUI either, like SoftwareBitmap.)

**Comment by jevansaks** [comment]
But in the same way that String is mapped to HSTRING, could we not map System.Drawing.Color to Windows.UI.Color? Is the conversion too lossy?

**Comment by dotMorten** [comment]
I find that a little weird too. I assume my question would apply to c++ as well so not just a question of language projection.
Also weird for Uno to have Window.UI.* as a first class type in an x-plat API.

It would also make the story much clearer to change everything under the Windows.UI namespace when moving to WinUI. It is UI after all

**Comment by billhenn** [comment]
Sorry to comment on such an old issue but now that I'm digging into WinUI, this is also something that was confusing/annoying to come across.  There are several pairs of types, things like `Color` / `Colors` and `FontWeight` / `FontWeights` where it feels odd to have one type in a `Windows.` namespace and another in a `Microsoft.` namespace.  

I get why some of those types might not have been moved (due to use elsewhere outside of WinUI) but it has a bad smell about it when looking at the API the first time.

**Comment by AtosNicoS** [comment]
Hey guys,
I just wanted to write a new WinUI 3 Application (1.0.0 release) and noticed, that specifying a ResourceDirectory with a simple color wont work. You explicitely need to add a using for that. That feels weird. Is that related to this issue?

```xaml
<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
					xmlns:ui="using:Windows.UI"
					mc:Ignorable="d">
<ui:Color x:Key="MyBlackColor">#FF2C2C2C</ui:Color>
```

**Comment by dotMorten** [comment]
@AtosNicoS Are you sure? I have this in my resource dictionary and works just fine in WinUI 3 v1.0:
```xml
<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
    <Color x:Key="Background0Color">#353535</Color>
</ResourceDictionary>
 ```

**Comment by AtosNicoS** [comment]
Yes I am sure. Your minimal example also does not work. However I noticed, that the code will still compile. Just the intellisense is not sure which type I am using. Maybe this is also related to resharper that I am using.

This is how it looks:
![grafik](https://user-images.githubusercontent.com/37442184/160769750-889514f0-c5d8-4fcf-bfb8-6f32f2f1265b.png)

The first works, but is marked red, the second fails to compile, the last also works and has proper intelli sense.
